specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-10-01-preview/BulkActions.json

```diff
@@ -0,0 +1,652 @@
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligenceBulkActions": {
+        "get": {
+          "x-ms-examples": {
+            "Get all bulk actions.": {
+              "$ref": "./examples/bulkActions/GetBulkActions.json"
+            }
+          },
+          "tags": [
+            "BulkActions"
+          ],
+          "description": "Gets all bulk actions.",
+          "operationId": "BulkActions_List",
+          "parameters": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+            },
+            {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "../../../common/2.0/types.json#/parameters/WorkspaceName"
+            },
+            {
+              "$ref": "../../../common/2.0/types.json#/parameters/ODataFilter"
+            },
+            {
+              "$ref": "../../../common/2.0/types.json#/parameters/ODataOrderBy"
+            },
+            {
+              "$ref": "../../../common/2.0/types.json#/parameters/ODataTop"
+            },
+            {
+              "$ref": "../../../common/2.0/types.json#/parameters/ODataSkipToken"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/BulkActionList"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/threatIntelligenceBulkActions/{bulkActionId}": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=8&length=13)

The top-level resource 'BulkAction' does not have list by subscription operation, please add it.